diwako_cbrn.vr/scripts/cbrn/functions/fn_loadoutEH.sqf

```diff
@@ -11,7 +11,12 @@ if (!(_unit getVariable ["cbrn_mask_on", false]) && {_hasMask}) then {
     cbrn_mask_abberation ppEffectEnable true;
     cbrn_mask_abberation ppEffectAdjust [0.005,0.005,true];
     cbrn_mask_abberation ppEffectCommit 1;
-
+	
+	if(cbrn_foggingEnabled && isNil "cbrn_fogPfh") then {
+		"cbrn_gasmask_fog" cutRsc ["cbrn_fog", "PLAIN", 0, false];
+		cbrn_fogPfh = [cbrn_fnc_fogPFH, 0.05, [cba_missiontime]] call CBA_fnc_addPerFrameHandler;
+	};
```

Comment:
Offload to post init. no need to repeatedly check if the pfh is running if you wan tit to run at all times when ever you reload or pick up a bandage (when fogging is enabled)

Then you also do not need to check if `cbrn_fogPfh` is nil. still write the variable tho, because people can use it when they want to for what ever reason they want.